pkgs/applications/office/jabref/default.nix

```diff
@@ -121,6 +124,7 @@ stdenv.mkDerivation rec {
     wrapProgram $out/bin/JabRef \
       --prefix PATH : ${lib.makeBinPath [ xdg-utils ]} \
       --set JAVA_HOME "${jdk}" \
+      --prefix XDG_DATA_DIRS : "$GSETTINGS_SCHEMAS_PATH" \
```

Comment:
I am sorry, I am confused by this. Isn't the value of the variable `GSETTINGS_SCHEMAS_PATH` evaluated by bash at this point? Where is this value coming from?

Reply:
It set by glib hook.

https://github.com/NixOS/nixpkgs/blob/eb642f80f9aecc19312909e08601a3c2020b5ce2/pkgs/development/libraries/glib/setup-hook.sh#L24

Reply:
Thanks.

So, I think then that it would be enough to add `glib` to the `buildInputs` instead of full `gtk3`?

Reply:
Good point. I'll have a test.

Reply:
Glib is not enough. GTK3 is required.

Reply:
I am just worried that `gtk3` is too heavy of a dependency.

I see that `gtk3` propagates `gsettings-desktop-schemas`. Could you try that?

I don't have a Linux box to try myself.

Reply:
It doesn't work. The error message is `GLib-GIO-ERROR **: 16:32:09.647: Settings schema 'org.gtk.Settings.FileChooser' is not installed
Trace/breakpoint trap` so I thought gtk is required.

Reply:
Ok, it's fine then, sorry for insisting.